tmail-backend/jmap/extensions-cassandra/src/main/scala/com/linagora/tmail/james/jmap/label/CassandraLabelRepository.scala

```diff
@@ -48,5 +48,10 @@ case class CassandraLabelRepositoryModule() extends AbstractModule {
 
     bind(classOf[LabelRepository]).to(classOf[CassandraLabelRepository])
     bind(classOf[CassandraLabelRepository]).in(Scopes.SINGLETON)
+
+    // Must be changed to CassandraLabelChangeRepository when CassandraLabelChangeRepository will be implemented
+    // https://github.com/linagora/tmail-backend/issues/715
+    bind(classOf[LabelChangeRepository]).to(classOf[MemoryLabelChangeRepository])
+    bind(classOf[MemoryLabelChangeRepository]).in(Scopes.SINGLETON)
```

Comment:
This would be dangerous IMO.
E.g: even with no guice binding for `CassandraLabelChangeRepository`, the distributed JMAP tests still pass. That would deceive us that no more Guice binding is needed.

Anyway either you or me will need to pay attention to this.

Reply:
Noted at https://github.com/linagora/tmail-backend/issues/715

If we don't have a bean for LabelChangeRepository, the distributed test for LabelSetMethod will be failed

Reply:
Well that just got merged :) so you can rebase and fix this now ;)